### PR TITLE
deflake: combine split metric point values during comparison

### PR DESCRIPTION
**Pull request by kyessenov**
Signed-off-by: Kuat Yessenov <kuat@google.com>

Sum up metrics that got split into separate points for whatever reason.

### CODE REVIEW

File: test/envoye2e/stackdriver_plugin/stackdriver.go [hunk 1]
```diff
@@ -120,11 +128,16 @@ func (sd *Stackdriver) Cleanup() {
...
 func (sd *Stackdriver) Check(p *driver.Params, tsFiles []string, lsFiles []SDLogEntry, verifyLatency bool) driver.Step {
 	// check as sets of strings by marshaling to proto
-	twant := make(map[string]struct{})
+	twant := make(map[string]int64)
 	for _, t := range tsFiles {
 		pb := &monitoring.TimeSeries{}
 		p.LoadTestProto(t, pb)
-		twant[prototext.Format(pb)] = struct{}{}
+		if len(pb.Points) != 1 || pb.Points[0].Value.GetInt64Value() == 0 {
+			panic("malformed metric golden")
```
Comment (douglas-reid) [inline test/envoye2e/stackdriver_plugin/stackdriver.go:136]: should we have a `ValidateGolden` pre-step that can fail (t.Fatalf) tests without panics as a better option here?

Comment (kyessenov) [reply test/envoye2e/stackdriver_plugin/stackdriver.go:136]: Seems like too much work for just one extension. I think panics will be caught early anyways.

Comment (douglas-reid) [reply test/envoye2e/stackdriver_plugin/stackdriver.go:136]: could we `log.Fatalf` here instead?  i know that doesn't make a ton of difference in the end, but I have a strong aversion to seeing `panic` in code (for better or worse).

Comment (kyessenov) [reply test/envoye2e/stackdriver_plugin/stackdriver.go]: Done.

File: Makefile.core.mk [hunk 2]
```diff
@@ -21,7 +21,7 @@ BAZEL_TARGETS ?= //...
 # Don't build Debian packages and Docker images in tests.
 BAZEL_TEST_TARGETS ?= ${BAZEL_TARGETS}
 E2E_TEST_TARGETS ?= $$(go list ./...)
-E2E_TEST_FLAGS ?=
+E2E_TEST_FLAGS ?= -p=1 -parallel=1
```
Comment (kyessenov) [inline Makefile.core.mk:24]: Golang log muxing is absolutely atrocious. Making it single threaded so we can actually parse the logs.

**Review by douglas-reid — APPROVED** [review]

**Pull request merged**